man/nvim.1

```diff
@@ -0,0 +1,388 @@
+.Cm errors.err ) .
+Further errors can be jumped to with the
+.Ic :cnext
+command.
+See
+.Ic :help quickfix .
+.It There are a number of other options:
+.It Fl -
+Interpret all further arguments as files.
+Can be used to edit files starting with a hyphen.
+.It Fl -literal
+Interpret filenames literally, that is do not expand wildcards.
+Has no effect on UNIX-like systems, where the shell expands wildcards.
+.It Fl e
+Ex mode.
+.It Fl E
+Improved Ex mode.
+.It Fl s
+Silent mode.
+Only takes effect if
+.Fl e
+is specified before
+.Fl s .
+.It Fl d Op Ar file ...
+Diff mode.
+Show the difference between two to four files.
+If
+.Ar file
+is omitted, options related to diff mode are still set.
+See
+.Ic :help diff .
+.It Fl R
+Read-only mode.
+Sets the option 'readonly'.
+Implies
+.Fl n .
+Buffers can still be edited, but cannot be written to disk if already
+associated with a file.
+To overwrite a file, add an exclamation mark to the needed Ex command, such as
+.Ic :w! .
+See
+.Ic :help 'readonly' .
+.It Fl Z
+Restricted mode.
+Disable commands that make use of an external shell.
+.It Fl m
+Disable file modifications.
+Unsets the option 'write'.
+Writing to a file is disabled, but buffers can still be modified.
+.It Fl M
+Disable file and buffer modifications.
+Unsets the options 'write' and 'modifiable'.
+Note that these options can be set to re-enable making modifications.
+.It Fl b
+Binary mode.
+See
+.Ic :help edit-binary .
+.It Fl l
+Lisp mode.
+Sets the options 'lisp' and 'showmatch'.
+.It Fl A
+Arabic mode.
+Sets the option 'arabic'.
+.It Fl F
+Farsi mode.
+Sets the options 'fkmap' and 'rightleft'.
+.It Fl H
+Hebrew mode.
+Sets the options 'hkmap' and 'rightleft'.
+.It Fl V Ns Oo Ar N Oc Ns Op Ar file
+Verbose.
+Print messages about which files are being sourced and for reading and
+writing an nviminfo file.
+.Ar N
+is the value for the 'verbose' option; defaults to
+.Cm 10
+if omitted.
+If
+.Ar file
+is specified, append messages to
+.Ar file
+instead of printing them.
+.It Fl D
+Debugging mode.
+Started when executing the first command from a script.
+.It Fl n
+Disable the use of swap files.
+Sets the option 'updatecount' to
+.Cm 0 .
+Can be useful for editing file(s) on a slow medium.
+.It Fl r , L
+List swap files with information about recovering them.
+.It Fl r Ar file
+Recovery mode.
+The swap file
+.Ar file
+is used to recover a crashed session.
+The swap file has the same filename as the text file, but with ".swp" appended.
+See
+.Ic :help recovery .
+.It Fl u Ar nvimrc
+Use
+.Ar nvimrc
+instead of the default of
+.Pa ~/.nvimrc .
+If
+.Ar nvimrc
+is
+.Cm NONE ,
+do not read any initialization files or environment variables.
+See
+.Ic :help initialization .
+.It Fl i Ar nviminfo
+Use
+.Ar nviminfo
+instead of the default of
+.Pa ~/.nviminfo .
+If
+.Ar nviminfo
+is
+.Cm NONE ,
+do not read or write an nviminfo file.
+See
+.Ic :help viminfo .
+.It Fl -noplugin
+Skip loading plugins.
+Implied by
+.Cm -u NONE .
+.It Fl o Ns Op Ar N
+Open
```

Comment:
Neither this nor the help say what happens when N is less than the number of file arguments given (it gives the first N files windows or tabpages and hides the rest). Just as a heads up.